CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Releases
 
+## Version 0.11.0
+
+Date: 2021-03-01
+
+The 0.11.0 release brings a number of exciting new features to Panel
+as well as some enhancements and bug fixes. This release is also
+required for Bokeh 2.3 compatibility since a lot of changes to the
+Bokeh property system required updates. Many thanks to the many
+contributors to this release including @MarcSkovMadsen, @xavArtley,
+@hyamanieu, @cloud-rocket, @kcpevey, @kaseyrussell, @miliante,
+@AjayThorve and @philippjfr.
+
+Major features:
+
+- A `Perspective` pane based on the FINOS Perspective library ([#2034](https://github.com/holoviz/panel/issues/2034))
+- Implement `--autoreload` functionality for the Panel server ([#1983](https://github.com/holoviz/panel/issues/1983))
+- Add `--warm` option to panel serve ([#1971](https://github.com/holoviz/panel/issues/1971)) 
+- Add ability to define JS modules and Template specific resources ([#1967](https://github.com/holoviz/panel/issues/1967))
+- Enables serving files and bokeh apps via `panel.serve` ([#1939](https://github.com/holoviz/panel/issues/1939)) 
+- Add a `Trend` indicator ([#1895](https://github.com/holoviz/panel/issues/1895))
+- Add `TextToSpeech` widget ([#1878](https://github.com/holoviz/panel/issues/1878))
+- Add `SpeechToText` Widget ([#1880](https://github.com/holoviz/panel/issues/1880))
+- Add `loading` parameter and spinners to all components ([#1730, [#2026](https://github.com/holoviz/panel/issues/2026))
+- Add `IDOM` pane ([#2004](https://github.com/holoviz/panel/issues/2004))
+- Add `Tabulator` widget ([#1531](https://github.com/holoviz/panel/issues/1531), [#1887](https://github.com/holoviz/panel/issues/1887)) 
+
+Enhancements:
+
+- Add watch argument to `bind` function ([#2000](https://github.com/holoviz/panel/issues/2000))
```

Comment:
```suggestion
- Add watch argument to `bind` function so that covers all the features of pn.depends ([#2000](https://github.com/holoviz/panel/issues/2000))
```